gbfs.md

```diff
@@ -241,19 +241,19 @@ Field Name | Required | Type | Defines
-&emsp;- `store_uri` *(beta)* | Conditionally Required | URI | URI where the rental iOS app can be downloaded from. Typically this will be a URI to an app store such as the Apple App Store. If the URI points to an app store such as the Apple App Store, the URI should follow iOS best practices so the viewing app can directly open the URI to the native app store app instead of a website. <br><br>If a `rental_uris`.`ios` field is populated then this field is required, otherwise it is optional. <br><br>See the [Analytics](#Analytics) section for how viewing apps can report the origin of the deep link to rental apps. <br><br>Example value: `https://apps.apple.com/app/apple-store/id123456789`
-&emsp;- `discovery_uri` *(beta)* | Conditionally Required | URI | URI that can be used to discover if the rental iOS app is installed on the device (e.g., using [`UIApplication canOpenURL:`](https://developer.apple.com/documentation/uikit/uiapplication/1622952-canopenurl?language=objc)). This intent is used by viewing apps prioritize rental apps for a particular user based on whether they already have a particular rental app installed. <br><br>This field is required if a `rental_uris`.`ios` field is populated, otherwise it is optional. <br><br>Example value: `com.abcrental.ios://`
+`license_id` <br/>*(added in v2.0-RC)* | Conditionally required | String | Required if the dataset is provided under a standard license. An identifier for a standard license from the [SPDX License List](https://spdx.org/licenses/). Provide `license_id` rather than `license_url` if the license is included in the SPDX License List. See the GBFS wiki for a [comparison of a subset of standard licenses](data-licenses.md). If the license_id or license_url fields are blank or omitted, this indicates that the feed is provided under the [Creative Commons Universal Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/legalcode).
+`license_url` | Conditionally required <br/>*(as of v2.0-RC)* | URL | Required if the dataset is  provided under a customized license. A fully qualified URL of a page that defines the license terms for the GBFS data for this system. Do not specify a `license_url` if `license_id` is specified. If the license_id or license_url fields are blank or omitted, this indicates that the feed is provided under the [Creative Commons Universal Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/legalcode). *(as of v2.0-RC)*
+`attribution_organization_name` <br/>*(added in v2.0-RC)* | Optional | If the feed license requires attribution, name of the organization to which attribution should be provided.
+`attribution_url` <br/>*(added in v2.0-RC)* | Optional | URL of the organization to which attribution should be provided.
```

Comment:
```suggestion
`attribution_url` <br/>*(added in v2.0-RC)* | Optional | URL | URL of the organization to which attribution should be provided.
```

Reply:
Thanks! I fixed in https://github.com/NABSA/gbfs/pull/221/commits/8d7c56ef46905d7594783ea70e888b45f9f1a244.